Projekty/Projekt1/AdriannaGrudzień_KingaUłasik/WUM_finalny_model.ipynb

```diff
@@ -0,0 +1,1742 @@
+{
```

Comment:
<p>Czy to jest potrzebne? Z tych pojęć znam tylko AIC i BIC i napisanie: "kompromis między dopasowaniem modelu a złożonością modelu. Niższa wartość AIC lub BIC oznacza lepsze dopasowanie. " jest nieprecyzyjne</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>